command/secrets/init/params.go

```diff
@@ -9,26 +9,32 @@ import (
 )
 
 const (
-	dataDirFlag = "data-dir"
-	configFlag  = "config"
-	ecdsaFlag   = "ecdsa"
-	blsFlag     = "bls"
-	networkFlag = "network"
-	numFlag     = "num"
+	dataDirFlag        = "data-dir"
+	configFlag         = "config"
+	ecdsaFlag          = "ecdsa"
+	blsFlag            = "bls"
+	networkFlag        = "network"
+	numFlag            = "num"
+	insecureLocalStore = "insecure"
```

Comment:
Nitpick: For the sake of consistency this should be suffixed with `Flag`

Reply:
Good catch 🎖️ Fixed in f5e00eb2a7031ed58278d18adcfb8bce4e15b812